sc/src/main/webapp/WEB-INF/jsp/tiles/views/home/experiment-list-latest.jsp

```diff
@@ -16,19 +16,9 @@
                 <c:forEach items="${latestExperiments}" var="experimentInfo">
                     <li>
                         <div class="media-object stack-for-small">
-                            <div class="media-object-section middle hide-for-small-only">
-                                <c:choose>
-                                    <c:when test="${experimentInfo.experimentType.baseline}">
-                                        <span data-tooltip class="baseline button-rd" title="Baseline experiment">B</span>
-                                    </c:when>
-                                    <c:otherwise>
-                                        <span data-tooltip class="differential button-rd" title="Differential experiment">D</span>
-                                    </c:otherwise>
-                                </c:choose>
-                            </div>
                             <div class="media-object-section middle hide-for-small-only">
                                 <s:eval expression="T(java.text.NumberFormat).getInstance().format(experimentInfo.numberOfAssays)" var="fmtNumberOfAssays" />
-                                <span class="button secondary no-action">${fmtNumberOfAssays}</span>
+                                <span class="button secondary no-action fixed-width-medium">${fmtNumberOfAssays}</span>
```

Comment:
A slightly better approach to this would be to use a grid rather than a list. The buttons could take up 20-25% of the width, and the title the rest. Though getting that to be nicely aligned and play nicely would be a massive pain. However it would remove the fixed width, which is generally not ideal.

Reply:
Yes, I agree, but I chose `media-object` becuase In large displays three columns is too wide and two columns is too narrow. Let’s wait for the EBI VF to include [the XY grid](https://foundation.zurb.com/sites/docs/xy-grid.html).